external/llvm-project/mlir/test/Integration/GPU/ROCM/gpu-to-hsaco.mlir

```diff
@@ -11,23 +11,23 @@
 func @other_func(%arg0 : f32, %arg1 : memref<?xf32>) {
   %c0 = constant 0 : index
   %c1 = constant 1 : index
-  %block_dim = dim %arg1, %c0 : memref<?xf32>
+  %block_dim = memref.dim %arg1, %c0 : memref<?xf32>
```

Comment:
Some dialect movements.  These tests don't run unless -DMLIR_INCLUDE_INTEGRATION_TESTS=ON.  I found them by accident when I put my resnet50 whole-model test in with the integration tests.